cpu/stm32f4/periph/cpuid.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2014 FU Berlin
```

Comment:
I did. Does the date only need to be updated or should the location be changed also?